app/utilities/drawing/master.py

```diff
@@ -0,0 +1,161 @@
+import sys
+import os
+import PIL
+from PIL import Image, ImageDraw, ImageOps, ImageFont, ImageEnhance, ImageFilter
+from pathlib import Path
+import requests
+from io import BytesIO
+
+from .utils import *
+from . import constants
+
+# changing import path
+abspath = os.path.abspath(__file__)
+ipath = os.path.dirname(abspath)
+os.chdir(ipath)
```

Comment:
Don't do this. Unless I'm mistaken, this changes the working directory for the _entire_ application. Remove this line and move it's relativeness to whenever you fetch files or w/e.

```suggestion
```